testing/dbtest/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM postgres:13-alpine
```

Comment:
Our base supported PG version is 11, so probably it'd be good to start with that if possible as that way if we are using something incompatible with any new code we'll catch it in the tests.

Reply:
Yea I can make that change. I was also thinking that we could expand this to build multiple images so we could test against multiple versions.

It is also worth noting that most of the tests on `main` currently run against `postgres:latest`.

Reply:
👍 on expanding to run on multiple versions. When you say that most of the tests run on `postgres:latest`, where do you see that? Our old dockertest config specified version 11...was CI doing something different?

Reply:
Maybe "most" isn't accurate. It does look like the dockertest config would default to 11. But I did notice a number of `postgres:latest` containers getting created when I was monitoring docker and running the full test suite from `main`. I didn't track down where they were specifically coming from.